### PR TITLE
Summariser reports broken terms

### PR DESCRIPTION
**Pull request by ondenman**
Adds a list of any added/removed terms from the summary.

Example output:
Summary of changes in `foo/bar.json`:
### Added

No people added
### Removed

No people removed
### Name Changes

No name changes
### Additional Name Changes

No name changes
### Wikidata Changes

No changes
### Added

No organizations added
### Removed

No organizations removed
### Added
- `term/42`
- `term/88`
### Removed
- `term/101`

Closes https://github.com/everypolitician/everypolitician/issues/368

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -94,6 +94,32 @@ def organizations_added
   def organizations_removed
     before.organizations - after.organizations
   end
+
+  def broken_terms
+    broken_terms = after.events.select do |event|
+      event[:classification] == "legislative period" && (event[:id] =~ /^(term)\/?\d+/).nil?
```
Comment (tmtmtmtm) [inline app.rb]: I'm not sure this Regex is doing what you think it's doing. At least it's not clear why you're making the "/" optional (but not the "term").

Either way I think this is a fatally flawed approach, as we don't guarantee that term IDs will be in this format.

**Comment by tmtmtmtm** [comment]
@ondenman per the inplace comment, I think the whole 'broken terms' approach isn't going to work, as you can't rely on that format for terms. But I don't think you actually _need_ that to work, as simply showing the new and removed terms should suffice. So I'd suggest just dropping the 'broken terms' section entirely, leaving only that report.

**Comment by ondenman** [comment]
@tmtmtmtm revised description.

**Comment by ondenman** [comment]
Closing. History is a mess. Opening a new PR (https://github.com/everypolitician/review_changes/pull/27)

**Pull request closed without merging**